hack/exercism_local_tooling_webserver

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
```

Comment:
Wat's this file for?

Reply:
When the local tooling webserver was broken and the fix had not been merged.  I will removed it.